html/php-challenge/index.php

```diff
@@ -28,6 +28,131 @@
 	}
 }
 
+// ****** RTボタン **********************
+// RT作成
+if (isset($_REQUEST['retweet_id'])) {
+	$getAllPosts = $db->prepare('SELECT * FROM posts WHERE id=?');
+	$getAllPosts->execute(array(
+		$_REQUEST['retweet_id']
+	));
+	$allPosts = $getAllPosts->fetch();
+
+	$rtMessage = "RT " . $allPosts['message']; 
+	$likeData = $db->prepare('INSERT INTO posts SET message=?, member_id=?, retweet_origin_id=?, created=NOW()');
+	$likeData->execute(array(
+		$rtMessage,
+		$_SESSION['id'],
+		$_REQUEST['retweet_id']
+	));
+
+	header('Location: index.php'); exit();
+}
+
+// RT削除
+if (isset($_REQUEST['delete_rt_origin_id'])) {
+	$delete = $db->prepare(('DELETE FROM posts WHERE member_id =? AND retweet_origin_id=?'));
+	$delete->execute(array(
+		$_SESSION['id'],
+		$_REQUEST['delete_rt_origin_id']
+	));
+
+	header('Location: index.php'); exit();
+}
+
+function checkIfRtOrigin($id) {
+	global $db;
+
+	$getRow = $db->prepare('SELECT * FROM posts WHERE member_id=? and retweet_origin_id=?');
+	$getRow->execute(array(
+		$_SESSION['id'],
+		$id
+	));
+	$row = $getRow->fetch();
+	if($row !== false) {
+		return true;
+	} else {
+		return false;
+	}
```

Comment:
(修正不要)
こうも書けますがどちらがよいか意見が割れているようです
```suggestion
	return ($row !== false);
```